creator/Documents/CommandBlocks.md

```diff
@@ -25,127 +25,116 @@ In this guide, we will set up a command block chain to look for a certain type o
 
 #### Setting the map up
 
-While this can be done on any Minecraft world that a player may have, we will be using a flat world in order to take advantage of using lower values for the coordinates.
+While command blocks can be used in any Minecraft world, we will be using a flat world for a simple and clean environment.
 
 1. Launch Minecraft: Bedrock Edition.
 1. Select **Play**, select **Create New**, then select **Create New World**.
-1. In the world settings, set the world to **Flat World**.
-1. In the world settings, set **Show Coordinates** to **True**.
-1. In the world settings, set **Enabled Cheats** to **True**.
+1. In the world settings, set the world type to **Flat**.
+1. In the world settings, enable **Show Coordinates**.
+1. In the world settings, enable **Activate Cheats**.
 1. Select **Create**.
 
 ### How to get Command Blocks
 
-Since command blocks are not a craft-able block, they will need to be added via the Command Console.
+Command blocks can only be obtained by running a command.
```

Comment:
I changed this because the "added via the command console" is not really a familiar way of referring to running a command from chat.